daemon/cluster/executor/container/adapter.go

```diff
@@ -23,8 +23,8 @@ import (
 	"github.com/docker/docker/daemon"
 	"github.com/docker/docker/daemon/cluster/convert"
 	executorpkg "github.com/docker/docker/daemon/cluster/executor"
+	networkSettings "github.com/docker/docker/daemon/network"
```

Comment:
I'm wondering if this import name is right.

We already use `networkSettings` has a variable name in `container_operations.go` and it uses the `Settings` type from this package. Also, it exposes a bit more than just 'settings' (eg. `AttachmentStore`). Maybe we can find a better name?

Reply:
I'm wondering if a better approach could be to alias network types in `github.com/docker/docker/daemon/network`, and use that instead of api types here (and then use the default import name for the daemon's `network` pkg) 🤔

Reply:
Yeah, the aliasing is a bit messy; I picked `networkSettings` here as that was the most commonly used alias for this internal package.